app/services/concerns/xccdf/hosts.rb

```diff
@@ -48,7 +48,6 @@ def test_result_profile
     def inventory_host
       @inventory_host ||= ::HostInventoryAPI.new(
         @host_inventory_id,
-        report_host,
```

Comment:
This was the only usage of this method in our code - we should delete the method as well.

Reply:
Said and done